internal/context/ngap_build.go

```diff
@@ -258,6 +258,51 @@ func BuildPDUSessionResourceModifyRequestTransfer(ctx *SMContext) ([]byte, error
 	}
 }
 
+func BuildPDUSessionResourceModifyConfirmTransfer(ctx *SMContext, tunnel *UPTunnel) ([]byte, error) {
+	confirmTransfer := ngapType.PDUSessionResourceModifyConfirmTransfer{}
+
+	// QoS Flow Modify Confirm List
+	qosList := &confirmTransfer.QosFlowModifyConfirmList
+	for _, dataPath := range tunnel.DataPathPool {
+		if dataPath.Activated {
+			ANUPF := dataPath.FirstDPNode
+			DLPDR := ANUPF.DownLinkTunnel.PDR
+			if DLPDR.Precedence == 255 {
```

Comment:
Could you add a comment to explain why DL PDR with Precedence 255 should be ignored here?